cmd/podman/containers/rm.go

```diff
@@ -140,6 +140,9 @@ func removeContainers(namesOrIDs []string, rmOptions entities.RmOptions, setExit
 			}
 			if setExit {
 				setExitCode(r.Err)
+				if rmOptions.Force && registry.GetExitCode() == 1 {
```

Comment:
This should only happen if the error is container does not exist, and this is the only error returned.

Reply:
The problem with that is the NOEXIST errors are all different.

Reply:
How are they all different? The setExitCode() already take care of setting this, so all what needs to be done is check if --force is set